src/content/docs/dev/send-tokens/sui-its.mdx

```diff
@@ -0,0 +1,502 @@
+    CoinManagement<T> {
+        treasury_cap: option::some(treasury_cap),
+        balance: option::none(),
+        distributor: option::none(),
+        operator: option::none(),
+        flow_limit: flow_limit::new(),
+        dust: 0,
+    }
+}
+```
+
+See [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L200) for an example of how deploy a new capped manangers.
+
+
+#### Locked Management
+
+Locked Management types Create a new `CoinManagement` with a `Balance`. The stored `Balance` can be used to take and put coins. This manger type will lock the sent coin with ITS when the coin is sent out of Sui and unlock the coin when it sent back into the Sui ecosystem. This type of manager is very useful if Sui is the *home-chain* for your ITS integration and you are using wrapped tokens on other chains that derive back to the canonical token on Sui. Integrating ` new_locked` manager type involves calling the following factory function.
+
+```move
+public fun new_locked<T>(): CoinManagement<T> {
+    CoinManagement<T> {
+        treasury_cap: option::none(),
+        balance: option::some(balance::zero()),
+        distributor: option::none(),
+        operator: option::none(),
+        flow_limit: flow_limit::new(),
+        dust: 0,
+    }
+}
+```
+
+See [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L194) for an example of how deploy a new capped managers.
+
+
+### Give Coin
+
+When the coin is being sent into the Sui ecosystem, the Management module will trigger the `give_coin()` function. This function updates the incoming [flow limit](#flow-limit) and then either mints new coins (if the instance has a treasury capability) or withdraws coins from the internal balance. It returns the coin object that is ready to be transferred.
+
+```move
+public(module) fun give_coin<T>(self: &mut CoinManagement<T>, amount: u64, clock: &Clock, ctx: &mut TxContext): Coin<T> {
+    self.flow_limit.add_flow_in(amount, clock);
+    if (has_capability(self)) {
+        self.mint(amount, ctx)
+    } else {
+        coin::take(self.balance.borrow_mut(), amount, ctx)
+    }
+}
+```
+
+
+### Take Balance
+When the coin is being out of the Sui ecosystem, the Management module will trigger the `take_balance()` function. This function updates the [flow limit](#flow-limit) for an outgoing transfer and then either burns the coins (if minting capability is available) or merges the deducted amount into the stored balance. It returns the numeric amount (as a `u64`) that was taken.
+
+
+```move
+public(package) fun take_balance<T>(self: &mut CoinManagement<T>, to_take: Balance<T>, clock: &Clock): u64 {
+    self.flow_limit.add_flow_out(to_take.value(), clock);
+    let amount = to_take.value();
+    if (has_capability(self)) {
+        self.burn(to_take);
+    } else {
+        self.balance.borrow_mut().join(to_take);
+    };
+    amount
+}
+```
+
+### Set Flow Limit
```

Comment:
document the entry point in the `interchain_token_service` module instead, as that is the one operators can use.